### ISSUE TITLE
Add SRV Record to win_dns_record Module

### ISSUE THREAD

**Issue by CSimpiFoN** [ansible-collections/community.windows#78]
##### SUMMARY
It would be great to manage SRV records too on Windows DNS server with win_dns_record module

##### ISSUE TYPE
- Feature Idea

##### COMPONENT NAME
win_dns_record

##### ADDITIONAL INFORMATION
Extend the functionality of win_dns_module

```
- name: SRV example
  win_dns_record:
    name: "metrics.node"
    type: "SRV"
    value: "db1"
    port: "9100"
    zone: "example.com"
```

**Comment by sakar97** [comment]
Working on it.

**Comment by jborean93** [comment]
Was implemented by https://github.com/ansible-collections/community.windows/pull/112 and included in the 1.0.0 release of this collection.
